spring-boot-dependencies/pom.xml

```diff
@@ -273,6 +274,11 @@
 			</dependency>
 			<dependency>
 				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-starter-data-couchbase</artifactId>
+				<version>1.4.0.BUILD-SNAPSHOT</version>
```

Comment:
@snicoll have a doubt about this version number being relevant. should the version of a spring-boot couchbase starter be in sync with the spring data version number or the current spring boot version number?